bucket/lz4.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "1.9.3",
+    "description": "Fast LZ compression algorithm library - tool",
+    "homepage": "https://github.com/lz4/lz4",
+    "license": "GPL-2.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/lz4/lz4/releases/download/v1.9.3/lz4_win64_v1_9_3.zip",
+            "hash": "519cb3bfc4975cce8747b1ecba5053137b6ce2a77b653e4d2c63fae057985449"
+        },
+        "32bit": {
+            "url": "https://github.com/lz4/lz4/releases/download/v1.9.3/lz4_win32_v1_9_3.zip",
+            "hash": "4eb78a7374b0ccadd46d6a148b42257e56890b5a1cceaf4db40d543281ad037c"
+        }
+    },
+    "pre_install": "Remove-Item \"$dir\\*\" -Recurse -Force -Exclude lz4.exe, README.md, NEWS",
```

Comment:
Why delete rest of the folders? Are they not helpful (even if not directly required)?